packages/venia-concept/.buildpack/create.js

```diff
@@ -0,0 +1,175 @@
+                const toPackageScript =
+                    npmCli === 'yarn'
+                        ? name => pkgTpt.scripts[name]
+                        : name =>
+                              pkgTpt.scripts[name].replace(
+                                  /yarn run/g,
+                                  'npm run'
+                              );
+                scriptsToCopy.forEach(name => {
+                    pkg.scripts[name] = toPackageScript(name);
+                });
+
+                pkg.scripts.build =
+                    'yarn run clean && yarn run validate-queries && yarn run build:prod';
+                if (process.env.DEBUG_PROJECT_CREATION) {
+                    console.warn(
+                        'process.env.DEBUG_PROJECT_CREATION is true, so we will assume we are inside the pwa-studio repo and replace those package dependency declarations with local file paths.'
+                    );
+                    const workspaceDir = resolve(__dirname, '../../');
+                    fse.readdirSync(workspaceDir).forEach(packageDir => {
+                        const packagePath = resolve(workspaceDir, packageDir);
+                        if (!fse.statSync(packagePath).isDirectory()) {
+                            return;
+                        }
+                        let name;
+                        try {
+                            name = fse.readJsonSync(
+                                resolve(packagePath, 'package.json')
+                            ).name;
+                        } catch (e) {}
+                        if (!name) {
+                            return;
+                        }
+                        const [{ filename }] = JSON.parse(
+                            require('child_process').execSync(
+                                'npm pack --json',
+                                { cwd: packagePath }
+                            )
+                        );
+                        [
+                            'dependencies',
+                            'devDependencies',
+                            'optionalDependencies'
+                        ].forEach(depType => {
+                            if (pkg[depType] && pkg[depType][name]) {
+                                const localDep = `file://${resolve(
+                                    packagePath,
+                                    filename
+                                )}`;
+                                pkg[depType][name] = localDep;
+                                if (!pkg.resolutions) {
+                                    pkg.resolutions = {};
+                                }
+                                pkg.resolutions[name] = localDep;
+                            }
+                        });
+                    });
+                }
+
+                fse.writeJsonSync(targetPath, pkg, {
+                    spaces: 2
+                });
+            },
+            'package-lock.json': ({
```

Comment:
We don't have a `package-lock` maintained in this repo aside from the one in devdocs. Is this necessary?

Reply:
Yes, for the core repository. For people creating their own projects, we want them to use anything they want--and they can!